### PR TITLE
Add model demos for DLA (Pytorch)

### PR DESCRIPTION
**Pull request by ashokkumarkannan1**

### CODE REVIEW

File: model_demos/cv_demos/dla/utils/model.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
+from os.path import join
```
Comment (nvukobratTT) [inline model_demos/cv_demos/dla/utils/model.py:1]: We can't keep full model implementation here as is. We need to reference the source of, or find another source (e.g. transformers, torchhub, etc.)

File: model_demos/cv_demos/dla/utils/model.py [hunk 2]
```diff
@@ -0,0 +1,592 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
```
[the remark this answers is not in the document]
Comment (milank94) [reply model_demos/cv_demos/dla/utils/model.py:1]: Yes, fine to use code / models under open license. We just need to follow certain standards for how to properly include them. We are missing the core License file in the root of the repository. 

We will add that into `main` and then we can review any open requests.

[the remark this answers is not in the document]
Comment (nvukobratTT) [reply model_demos/cv_demos/dla/utils/model.py:1]: Great! Thanks for the info! 

Just to confirm, this change is OK to merge as is? After you add a Licence file to the root of the repository, we'll rebase the track_pybuda branch to the latest version. That should enable this model to be pushed with one of the following Alpha release?

[the remark this answers is not in the document]
Comment (milank94) [reply model_demos/cv_demos/dla/utils/model.py:1]: Yes, that should be fine. Good plan!

**Review by nvukobratTT — APPROVED** [review]

**Comment by nvukobratTT** [comment]
One detail that is missing, please update:
- model_demos/README.md
- model_demos/pyproject.toml

[Here](https://github.com/tenstorrent/tt-buda-demos/pull/61/files) is one sample of doing so. Feel free to cover this with new MR

**Pull request merged**